test/e2e/network_test.go

```diff
@@ -297,6 +298,23 @@ var _ = Describe("Podman network", func() {
 		Expect(rmAll).Should(ExitCleanly())
 	})
 
+	It("podman run container host interface name", func() {
+		Skip("We need to update netavark first.")
```

Comment:
please say `FIXME: We need netavark >= v1.14 for host interface support`.

This is important for anyone looking at this to know when this can be dropped.